singularity/code/i18n.py

```diff
@@ -168,6 +170,18 @@ def translate(string, *args, **kwargs):
 except ImportError:
     import __builtin__ as builtins
 
+
+def lex_sorting_form(name):
+    """For lexicographic sorting when languages use characters not in 7-bit ASCII. e.g. é or ü.
+
+    Use like this:
+
+    listdata.sort(key=lambda an_object: lex_sorting_form(an_object.name))"""
+
+    return unidecode(unicode(name)).lower()
+
+builtins.__dict__['lex_sorting_form'] = lex_sorting_form
```

Comment:
I would prefer that we just import the function as a regular function from i18n instead of injecting it into builtins. The injection breaks a lot of static analysis tooling/IDE highlighting (with `_ ` being a "historical" exception)

Reply:
Done